.clang-format

```diff
@@ -11,14 +11,14 @@
 
 ---
 BasedOnStyle: LLVM
-AlignConsecutiveMacros: AcrossComments
-AttributeMacros:
-  - __aligned
-  - __deprecated
-  - __packed
-  - __printf_like
-  - __syscall
-  - __subsystem
+#AlignConsecutiveMacros: AcrossComments
```

Comment:
this was added in clang-format 9

Reply:
`AlignConsecutiveMacros: AcrossComments` does not work with clang-format v10.

Reply:
that's weird docs clearly say it was added on version 9: https://clang.llvm.org/docs/ClangFormatStyleOptions.html, maybe:

```
AlignConsecutiveMacros:
  Enabled: true
  AcrossEmptyLines: false
  AcrossComments: true
```
?

Reply:
apparently older versions only support `AlignConsecutiveMacros: true`

Reply:
^ a point that makes me think shipping the right clang-format will be useful @stephanosio